src/main/java/net/mcreator/ui/minecraft/states/DefaultPropertyValue.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.states;
+
+import com.google.gson.*;
+import com.google.gson.annotations.JsonAdapter;
+
+import javax.annotation.Nonnull;
+import java.lang.reflect.Type;
+
+@JsonAdapter(DefaultPropertyValue.GSONAdapter.class) public record DefaultPropertyValue<T>(PropertyData<T> property,

```

Comment:
Indentation here should be fixed

Reply:
IDEA will fix it its own way, you see the result.